.github/workflows/release.yaml

```diff
@@ -37,7 +37,7 @@ jobs:
         run: go test -v ./...
 
       - name: Run GoReleaser
-        uses: goreleaser/goreleaser-action@0110a4acb73fda6a2c401b6d1cb70a11d5b0fadf
+        uses: goreleaser/goreleaser-action@v2
```

Comment:
I think this was explicitly set by @akshay196 to be a sha

Reply:
any specific reason @meain @akshay196-rafay?? but now `v2` works properly.

Reply:
Reference from https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#using-third-party-actions:
> Pinning an action to a full length commit SHA is currently the only way to use an action as an immutable release. Pinning to a particular SHA helps mitigate the risk of a bad actor adding a backdoor to the action's repository, as they would need to generate a SHA-1 collision for a valid Git object payload.

@rustiever You can use SHA of a commit from v2 release.

Reply:
updated @akshay196